.github/workflows/auto-label.yml

```diff
@@ -0,0 +1,30 @@
+name: Auto-Label for Write Access and Python Files
```

Comment:
This file checks if a PR is (1) created by a user with write access and (2) modifying at least 1 Python file. If both are true, it automatically adds the `gpuci` label to automatically kick off our GPU CI.

Reply:
Ah another thing: This didn’t work because using secrets.GITHUB_TOKEN is not permitted to trigger other workflows. GH disallows that to prevent spam. You would need to use your personal access token (PAT). However, since we’re in a public environment, you would need to limit that to a protected branch (via environments). It’s still doable (from branch main, react to all issue change events), but a bit more tricky. As long as the PAT is not exposed to all branches, this is fine

Reply:
Thanks! Added this comment to https://github.com/NVIDIA/NeMo-Curator/issues/259 for now.